src/StatsdClient/Serializer/EventSerializer.cs

```diff
@@ -47,7 +53,7 @@ public static string GetCommand(string title, string text, string alertType, str
                 result += string.Format(CultureInfo.InvariantCulture, "|t:{0}", alertType);
             }
 
-            result += MetricSerializer.ConcatTags(constantTags, tags);
+            result += MetricSerializer.ConcatTags(_constantTags, tags);
 
             if (result.Length > MaxSize)
```

Comment:
https://github.com/DataDog/dogstatsd-csharp-client/pull/124/commits/88643cf170dad14e90a706f117980fe0349ccb3a#diff-2b2429c2b7220221984fa0ab16b8924eL66: 
In the previous version, when truncating the metric, `GetCommand` overload was called without `constantTags` which was an issue.  As `_constantTags` is a member of EventSerializer, `constantTags` is always used in the new version.